app/javascript/mixins/mixin-responsive.js

```diff
@@ -6,8 +6,10 @@ export default {
       windowWidth: 0,
       currentBreakpoint: '',
       breakpoints: {
+        xs: 640,
         small: 720,
-        medium: 960,
+        medium: 1024,
+        large: 1708
```

Comment:
These numbers could be turned into constants e.g. outside the `export default` set `const BREAKPOINT_XS = 640`. I assume they only appear once, but for these kind of settings, it's good to turn them into constants

Reply:
There was conveniently already a constants file set up: `app/javascript/constants/constants-breakpoints.js`. The breakpoints stored there are identical to the ones used here in the mixin, except for the absence of the `xs` breakpoint, which I've now added in after checking this won't affect anything.

<img width="1307" alt="image" src="https://user-images.githubusercontent.com/56026099/126328423-82d64419-4962-46d2-aa02-b23a9c2f2825.png">

Reply:
I'll leave you to check this and resolve the conversation if you think this is okay :)

Reply:
Fantabulous